mandatory/3-playing-computer.js

```diff
@@ -8,11 +8,11 @@
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  3. What is printed to the console? 2,6,4,9,6,18,8
+  4. How many times is "f1" called? 2
+  5. How many times is "f2" called? 2
+  6. What value does the "a" parameter take in the first "f1" call? 6
```

Comment:
This is actually 0. The first call to f1 passes the 'i' variable as the a parameter. On the first iteration of the loop 'i' is 0.